config/cht/test/targets/births.spec.js

```diff
@@ -0,0 +1,44 @@
+const chai = require('chai');
+const expect = chai.expect;
+const path = require('path');
+const moment = require('moment')
+const sinon = require('sinon');
+const TestRunner = require('medic-conf-test-harness');
+const { newbornBaby } = require('../contacts');
+const harness = new TestRunner({
+  xformFolderPath: path.join(__dirname, '../../forms/app'),
+  inputs: { content: { contact: newbornBaby } }
+});
+let clock;
+
+describe('Births this month target tests', () => {
+  before(async () => {
+    return await harness.start();
+  });
+  after(async () => {
+    return await harness.stop();
+  });
+  beforeEach(async () => {
+    await harness.clear();
+  });
+  afterEach(() => {
+    expect(harness.consoleErrors).to.be.empty;
+    if (clock) clock.restore();
+  });
+
+  it('Birth this month should be counted', async () => {
+    //await harness.setNow('2000-04-30');//DOB: 2000-04-24
+    clock = sinon.useFakeTimers(moment('2000-04-30').toDate());
```

Comment:
I should fix this for you, `harness.setNow` does the same as `sinon.useFakeTimers` and you don't always need to call both....

Reply:
It only works for some of the tests. In this test file, using `harness.setNow` only works for the first test but not for the second (`birth last month should not be counted`). What could be the reason?